backend/src/auth/auth.spec.ts

```diff
@@ -2,97 +2,29 @@ import { Test, TestingModule } from '@nestjs/testing';
 import { AuthService } from './auth.service';
 import { AuthController } from './auth.controller';
 import { AuthGuard } from './auth.guard';
-import { JwtService } from '@nestjs/jwt';
-import { ConfigModule } from '@nestjs/config';
-import { TypeOrmModule } from '@nestjs/typeorm';
-import { TypeOrmConfigService } from '../typeorm/typeorm.service';
 
-import { InsertResult } from 'typeorm';
-import { AuthModule } from './auth.module';
-import { ChatModule } from '../chats/chat/chat.module';
-import { Game } from '../pong/game/entities/game.entity';
-import { GameInvite } from '../pong/game_invite/entities/game-invite.entity';
-import { GameInvitesModule } from '../pong/game_invite/game-invite.module';
-import { GameModule } from '../pong/game/game.module';
-import { MatchmakingRequest } from '../pong/matchmaking-request/entities/matchmaking-request.entity';
-import { MatchmakingRequestModule } from '../pong/matchmaking-request/matchmaking-request.module';
-import { MessageModule } from '../chats/message/message.module';
-import { RoleModule } from '../chats/role/role.module';
-import { SharedModule } from '../shared/shared.module';
-import { User } from '../users/user/entities/user.entity';
-import { UserChatModule } from '../chats/user-chat/user-chat.module';
-import { UserController } from '../users/user/user.controller';
-import { UserModule } from '../users/user/user.module';
-import { UserService } from '../users/user/user.service';
+import { AllTestingModule } from '../shared/test.module';
 
 describe('Auth', () => {
 	let authController: AuthController;
-	let userController: UserController;
 	let authService: AuthService;
-	let userService: UserService;
-
-	let users = [
-		{
-			name: 'u1',
-			password: 'p1',
-			password_confirm: 'p1',
-		},
-		{
-			name: 'u2',
-			password: 'p2',
-			password_confirm: 'p2',
-		},
-		{
-			name: 'u3',
-			password: 'p3',
-			password_confirm: 'p3',
-		},
-	];
```

Comment:
The mocking was removed cause it wasn't used for anything?

Reply:
indeed